tests/conftest.py

```diff
@@ -95,12 +102,14 @@ def app(request):
     with app.app_context():
         db.create_all()
         list(search.create(ignore=[400]))
-        sleep(5)
+        sleep(10)
+        current_oaiserver.unregister_signals_oaiset()
```

Comment:
Do we need to call it always? The signal registration should be guarded by config option.